src/graphql/resolvers.ts

```diff
@@ -14,5 +17,12 @@ export const resolvers = {
         token: token,
       };
     },
+    createUser: async (_: unknown, { name, email, birthDate, cpf, password }: CreateuserInterface, context) => {
+      if (!tokenIsValid(context['token'])) {
+        throw new CustomError('JWT inválido.', 401, 'Operação não autorizada.');
+      }
+      const { user } = await createUser({ name, email, birthDate, cpf, password });
+      return { user };
```

Comment:
Poderia retornar direto aqui:

```suggestion
      return createUser({ name, email, birthDate, cpf, password });
```